PolishWithBlondes/src/components/newsletter/Newsletter.tsx

```diff
@@ -19,7 +19,29 @@ export const Newsletter = () => {
           </p>
         </div>
         <div className="column right">
-          <img className="iphone-icon" src="src/assets/iphone-icon.png" />
+          <form>
+            <input
+              type="text"
+              id="fname"
+              name="fname"
+              placeholder="First Name"
+            ></input>
+            <br />
+            <input
+              type="text"
+              id="email"
+              name="email"
+              placeholder="Email"
+            ></input>
+            <br />
+            <p className="privacy-text">
+              By signing up, you'll receive communications from Polish with
+              Blondes. For more information, see our{' '}
+              <a href="#">privacy policy</a>.
+            </p>
+            <br />
```

Comment:
There is no point of using <br/> here.
In 99% of use cases if you want to add a space between two elements, just use css rules like:
display: block
margin or padding. 
Please remove all <br/> tags here